src/components/v2/MarkdownRender.vue

```diff
@@ -0,0 +1,99 @@
+<script lang="ts" setup>
+import { computed } from 'vue';
+import markdownit from 'markdown-it';
```

Comment:
I'm not 100% certain how experimental this is, but the new website uses an undocumented endpoint, e.g. `https://thunderstore.io/api/cyberstorm/package/ymc_mhz/morehead/latest/readme/` (also available for changelog). The major thing is it returns HTML instead of MD, so we could avoid requiring these dependencies, and IIRC get some security against XSS at the same time. Additionally it would make it easier to have the content look the same on the manager and the website, so the author doesn't need to specifically target one or another.

Also Oksamies might be able to provide some CSS that would make it even easier to get the content look the same, but I'm not 100% sure how feasible that is.